src/core/tests/test_schema.py

```diff
@@ -0,0 +1,51 @@
+import schemathesis
+from hypothesis import settings, HealthCheck
+
+schema = schemathesis.from_uri("http://127.0.0.1:5000/api/v1/doc/swagger.json")
+
+
+# @schema.parametrize(endpoint="/api/v1/analyze")
+# @settings(max_examples=100, suppress_health_check=[HealthCheck.filter_too_much])
+# def test_schema_analyze(case, access_token):
+#     case.call_and_validate(headers={"Authorization": access_token})
+
+
+@schema.parametrize(endpoint="/api/v1/assess")
+def test_schema_assess(case, access_token):
+    case.call_and_validate(headers={"Authorization": access_token})
```

Comment:
I think we need a custom authenticator:

https://schemathesis.readthedocs.io/en/stable/auth.html#custom-authentication

Otherwise, wouldn't it be necessary to add each endpoint?

Reply:
access_token fixture should handle that